Lib/test/test_getpass.py

```diff
@@ -162,5 +162,66 @@ def test_falls_back_to_stdin(self):
             self.assertIn('Password:', stderr.getvalue())
 
 
+try:
+    import msvcrt
+except ImportError:
+    msvcrt_available = False
+else:
+    msvcrt_available = True
```

Comment:
Can we have that import at the very top of the file with other imports? we don't usually have imports in the middle of a module. In addition, we usually have the pattern:

```py
try:
	import module
except ImportError:
	module = None
```

and then do `skipIf(module is None, ...)`